pkg/image/api/types.go

```diff
@@ -60,6 +62,92 @@ type ImageLayer struct {
 	Size int64
 }
 
+const (
+	// The supported type of image signature.
+	ImageSignatureTypeAtomic string = "atomic"
```

Comment:
Renamed to `|AtomicImageV1`.|
On 06/06/2016 07:33 PM, Clayton Coleman wrote:
>
> In pkg/image/api/types.go <https://github.com/openshift/origin/pull/8371#discussion_r65932844>:
>
> > @@ -60,6 +62,92 @@ type ImageLayer struct {
> >  	Size int64
> >  }
> >
> > +const (
> > +	// The supported type of image signature.
> > +	ImageSignatureTypeAtomic string = "atomic"
>
> I want this more specific. This should be a constant (like all our other types) and be appropriately versioned. So 
> |AtomicImageV1| is acceptable, or I could possibly accept |v1.projectatomic.io/Image|.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/openshift/origin/pull/8371/files/a36c2e3016ed8fe8497e02a4d32e424450596376#r65932844>, or mute the 
> thread <https://github.com/notifications/unsubscribe/ABaQk_3p7Udyasn2gg5YKkAKuY35AH8Uks5qJFn7gaJpZM4IBAFQ>.
>